package.json

```diff
@@ -135,7 +135,6 @@
     "invariant": "^2.2.4",
     "keytar": "7.6.0",
     "lunr": "^2.3.9",
-    "minilog": "https://github.com/cozy/minilog.git#master",
```

Comment:
ok to remove this one, but you have to add @cozy/minilog. No?

Reply:
oh, already there. Sorry

Reply:
the real fix should have been to remplace it by cozy-minilog instead of @cozy/minilog